warehouse/tasks.py

```diff
@@ -195,7 +195,10 @@ def includeme(config):
 
     broker_transport_options = {}
 
-    broker_url = s["celery.broker_url"]
+    broker_url = s.get("celery.broker_url")
+    if broker_url is None:
+        broker_url = s["celery.broker_redis_url"]
```

Comment:
I'm guessing that this will also be paired with removing `BROKER_URL` envvar from the runtime as part of deployment?

Reply:
correct, it would need to be.